forge-gui/res/cardsfolder/upcoming/genku_future_shaper.txt

```diff
@@ -0,0 +1,12 @@
+Name:Genku, Future Shaper
+ManaCost:2 W U
+Types:Legendary Creature Moonfolk Wizard
+PT:2/5
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Permanent.YouCtrl+nonToken | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ Whenever another nontoken permanent you control leaves the battlefield, choose one that hasn't been chosen this turn. Create a creature token with those characteristics.
+SVar:TrigCharm:DB$ Charm | ChoiceRestriction$ ThisTurn | Choices$ DBToken1,DBToken2,DBToken3
+SVar:DBToken1:DB$ Token | TokenAmount$ 1 | TokenScript$ w_2_2_fox_vigilance | TokenOwner$ You | SpellDescription$ 2/2 white Fox with vigilance.
+SVar:DBToken2:DB$ Token | TokenAmount$ 1 | TokenScript$ u_1_2_moonfolk_flying | TokenOwner$ You | SpellDescription$ 1/2 blue Moonfolk with flying.
+SVar:DBToken3:DB$ Token | TokenAmount$ 1 | TokenScript$ b_1_1_rat_lifelink | TokenOwner$ You | SpellDescription$ 1/1 black Rat with lifelink.
+A:AB$ PutCounterAll | Cost$ 3 W U | ValidCards$ Creature.YouCtrl | ValidCardsDesc$ creature you control | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on each creature you control.
+DeckHas:Ability$Counters|Token
```

Comment:
|LifeGain & Types$Fox|Moonfolk|Rat & Color$Black

Reply:
Actually the Type part might not be needed thanks to tool4ever latest PR

Reply:
Added `LifeGain` ability tag